src/App.js

```diff
@@ -127,6 +128,9 @@ function App() {
             )}
           </div>
         </div>
+        <div>
```

Comment:
@swimmingHaribo I think we can add a button to show the news, instead of always showing it, as UI is not looking good like that.